webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -50,6 +50,12 @@
   >
     <div class="page-heading">
       <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signature }} ]</h2>
+      <nav>
+        <ul class="options">
+          <li><a class="button" href="https://support.mozilla.org/search?{{ make_query_string(q=report.signature) }}" title="Find more answers at support.mozilla.org!">Search Mozilla Support for this signature</a></li>
+          <li><a class="button" href="https://developer.mozilla.org/en-US/docs/Understanding_crash_reports" title="MDN documentation about crash reports">How to read this crash report</a></li>
+        </ul>
+      </nav>
```

Comment:
I moved these buttons upwards into the header to match what we do with Super Search.